docs/integrations/sources/chartmogul.md

```diff
@@ -1,46 +1,61 @@
 # Chartmogul
-This page contains the setup guide and reference information for the Chartmogul source connector.
+This page contains the setup guide and reference information for the [Chartmogul](https://chartmogul.com/) source connector.
 
 ## Prerequisites
-* API key
-* Start date
-* Interval
+- A Chartmogul API Key.
+- A desired start date from which to begin replicating data.
+- A desired interval period for the `CustomerCount` stream. The available options are **day**, **week**, **month**, and **quarter**.
 
 ## Setup guide
-### Step 1: Set up Chartmogul
-
-1. To get access to the Chartmogul API you need to create an API key, please follow the instructions in this [documentation](https://help.chartmogul.com/hc/en-us/articles/4407796325906-Creating-and-Managing-API-keys#creating-an-api-key).
+### Step 1: Set up a Chartmogul API key
+1. Log in to your Chartmogul account.
+2. In the left navbar, select **Profile** > **View Profile**.
+3. Select **NEW API KEY**.
+4. In the **Name** field, enter a unique name for the key.
+5. If you are a Staff, Admin, or Owner, set the **Access Level** to either **Read-only** or **Read & Write** using the dropdown menu.
+6. Click **ADD** to create the key.
+7. Click the **Reveal** icon to see the key, and the **Copy** icon to copy it to your clipboard.
+
+For further reading on Chartmogul API Key creation and maintenance, please refer to the official 
+[Chartmogul documentation](https://help.chartmogul.com/hc/en-us/articles/4407796325906-Creating-and-Managing-API-keys#creating-an-api-key).
 
 ### Step 2: Set up the Chartmogul connector in Airbyte
-**For Airbyte Cloud:**
+1. [Log in to your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account, or navigate to the Airbyte Open Source dashboard.
+2. In the left navigation bar, click **Sources**.
+
+:::tip
+If this is your first time setting up an Airbyte source, skip the next step and proceed to step 4.
+:::
+
+3. In the top-right corner, click **+ New source**.
+4. Select **Chartmogul** from the list of available sources.
+5. Enter a **Source name** of your choosing.
+6. Enter the **API key** that you obtained.
+7. Enter a **Start date**. The **Start date** must be formatted as a UTC date and time in the following format: `yyyy-mm-ddThh:mm:ssZ`. For example, an input of `2017-01-25T06:30:00Z` will signify a start date of 6:30 AM UTC on January 25th, 2017. When feasible, any data before this date will not be replicated.
```

Comment:
Same comment as the other PR about start date, no need to talk about the format except the user is programmatic

Reply:
Fixed 👍